### ISSUE TITLE
kmail is broken on master

### ISSUE THREAD

**Issue by FRidh** [NixOS/nixpkgs#102167]
**Describe the bug**
Run-time fatal error.

**To Reproduce**
```
$ kmail
```
starts the program but results in a dialog that says
```
The Email program encountered a fatal error and will terminate now.
The error was:
Unable to obtain agent type ''.
```

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

Nixpkgs at d420c85a5593866939e5fe72ee9396bbf271bc92

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: `kmail`
# a list of nixos modules affected by the problem
module:
```

**Comment by FRidh** [comment]
https://github.com/NixOS/nixpkgs/pull/101369 gives build-time errors of dependencies

**Comment by doronbehar** [comment]
Indeed kdiagram is broken with qt515 and not with qt514.. but we will _not_ bring down all other kdeApplications and frameworks to qt514. The fix is a simple version bump (included in #101369) .
